doc/man/man3/intro.3monocypher

```diff
@@ -101,6 +101,11 @@ implements X25519, an elliptic curve Diffie Hellman key exchange
 algorithm based on Curve25519.
 X25519 derives a shared secret from two private/public key pairs.
 It is fast, simple, and relatively easy to implement securely.
+.Pp
+For specialized protocols that require indistinguishability from random
+noise,
+.Xr crypto_curve_to_hidden 3monocypher
+gives the option to disguise X25519 public keys as random noise.
```

Comment:
_Ephemeral_ public keys to be more precise. Ratchet keys can be encrypted with the current session key, and static keys are reused and re-transmitted, and as such cannot possibly be indistinguishable from random —not the second time you send them over the network. Which is why static keys should also be encrypted if at all possible.

Not sure we nevertheless want to add the word "ephemeral". It would be more accurate, it would drop a keyword people can search for, but it could also confuse them…

Reply:
How about “one-time use (ephemeral)” instead?

Reply:
Oh yes, _“one-time use (ephemeral)”_ is perfect.